src/invoker_cache.cpp

```diff
@@ -44,13 +45,27 @@ boost::optional<const Invoker&> InvokerCache::GetFound1_0(const std::string& net
                                                           const std::string& algorithm) const
 {
     const auto item = invokers.find(network_config);
-    if(item == invokers.end() || item->second.found_1_0.empty())
+    if(item == invokers.end())
+    {
+        MIOPEN_LOG_I2("No invokers found for " << network_config);
+        return boost::none;
+    }
+    if(item->second.found_1_0.empty())
+    {
+        MIOPEN_LOG_I2("Invokers found for " << network_config
+                                            << " but there is no find 1.0 result.");
         return boost::none;
+    }
     const auto& item_invokers = item->second.invokers;
     const auto& found_1_0_ids = item->second.found_1_0;
     const auto found_1_0_id   = found_1_0_ids.find(algorithm);
     if(found_1_0_id == found_1_0_ids.end())
+    {
+        MIOPEN_LOG_I2("Invokers found for " << network_config
+                                            << " but there is no one with an algorithm "
```

Comment:
the english here is a little ambiguous, are you trying to say:
Invoker found for n_config but there is no algorithm implemented?

Reply:
"but there is no one that is capable to run the requested algorithm among those."

Reply:
I mean that there are some invokers registered in the cache, but none of them have the specified algorithm.